scripts/release.js

```diff
@@ -125,6 +125,11 @@ async function main() {
   step('\nUpdating package version...')
   updateVersion(targetVersion)
 
+  if ('npm_package_scripts_lint' in process.env) {
```

Comment:
I don't like relying on envs such as these, perhaps:

```suggestion
  if (pkg.scripts.lint != null) {
```

Reply:
Wow, I didn't know that this is possible, I will test this out later
IMO this is a huge improvement 👍